createtable.go

```diff
@@ -158,6 +158,10 @@ func (ctb *CreateTableBuilder) SetFlavor(flavor Flavor) (old Flavor) {
 	return
 }
 
+func (ctb *CreateTableBuilder) GetFlavor() Flavor {
```

Comment:
The method name should be `Flavor` rather than `GetFlavor` according to [Effective Go](https://go.dev/doc/effective_go#Getters). And don't forget to add comment to describe this method.